<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLCDNumber" name="lcdDistance">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdSpeed">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>80</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="DistanceButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SpeedButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QuitButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>201</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ConnectButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>201</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DistanceSpeedButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>390</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance and Speed</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdDistance_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>240</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdSpeed_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>240</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearScreenButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>201</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Screens</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>909</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>DistanceButton</sender>
   <signal>clicked()</signal>
   <receiver>lcdDistance</receiver>
   <slot>setDecMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SpeedButton</sender>
   <signal>clicked()</signal>
   <receiver>lcdSpeed</receiver>
   <slot>setDecMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>736</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>740</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DistanceSpeedButton</sender>
   <signal>clicked()</signal>
   <receiver>lcdDistance_2</receiver>
   <slot>setDecMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>575</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DistanceSpeedButton</sender>
   <signal>clicked()</signal>
   <receiver>lcdSpeed_2</receiver>
   <slot>setDecMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>658</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>736</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
